llvm/lib/Target/AArch64/AArch64TargetTransformInfo.cpp

```diff
@@ -3277,13 +3276,15 @@ InstructionCost AArch64TTIImpl::getVectorInstrCostHelper(
         // important.
         UserToExtractIdx[U];
       }
+      if (UserToExtractIdx.empty())
```

Comment:
I don't see the link with the change in `IsExtractLaneEquivalentToZero`, is this related?